gdk/gdk.go

```diff
@@ -2334,23 +2292,20 @@ func (v *Window) PixbufGetFromWindow(x, y, w, h int) (*Pixbuf, error) {
 		return nil, nilPtrErr
 	}
 
-	obj := &glib.Object{glib.ToGObject(unsafe.Pointer(c))}
-	p := &Pixbuf{obj}
-	//obj.Ref()
-	runtime.SetFinalizer(p, func(_ interface{}) { obj.Unref() })
-	return p, nil
+	obj := glib.Take(unsafe.Pointer(c))
```

Comment:
`gdk_pixbuf_get_from_window` is marked as "transfer full". I do think that means we don't call `Ref()` but just ensure to clean up in a finalizer.

Reply:
Done